### DISCUSSION TITLE
Daily 20/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#119] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Dificultades o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve revisando la documentación del proyecto, agregando diagramas e información en la wiki e información y completando el readme. Agregué tareas a las issue, aunque no terminé con esto. Avancé un poco con el contenido de Angular.
Revisando las historias de usuario me di cuenta que no hay ninguna para la barra de navegación y el footer, así que habría que agregarlas.

Hoy continuaré agregando las tareas a las issue que faltan. Trataré de completar el capítulo de Angular ya que es lo que hay que hacer para el sprint que viene.

**Comment by rpgrca** [comment]
Hoy estuve bastante ocupado en el trabajo así que no hice mucho. Estuve en la oficina virtual a la noche y completé el formulario para registrarme. Mañana tengo dentista a la mañana y una ecografía a la tarde así que también voy a estar algo apretado de tiempo.

**Comment by BraianTroncoso** [comment]
Ayer que no pude pasarme a la Dailys, estuve un poco enfermo por ende solo hice unos ejercicios de Java para practicar mas OOP, y vi por encima un par de proyectos en Angular.
